src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrManagerBase.cs

```diff
@@ -60,11 +63,13 @@ public abstract class VmrManagerBase
         _localGitClient = localGitClient;
         _dependencyFileManager = dependencyFileManager;
         _fileSystem = fileSystem;
+
+        LocalVmr = localGitRepoFactory.Create(_vmrInfo.VmrPath);
     }
 
     public async Task<IReadOnlyCollection<VmrIngestionPatch>> UpdateRepoToRevisionAsync(
         VmrDependencyUpdate update,
-        NativePath clonePath,
+        ILocalGitRepo clone,
```

Comment:
A nit: `clone` -> `repoClone` to make it explicit at the place where the variable is used (even though the context is git repos).